scripts/spack.yaml

```diff
@@ -16,9 +16,9 @@ spack:
     - py-sphinx
     - py-sphinxcontrib-bibtex
     - py-sphinxcontrib-mermaid
-    - "root@6.24: cxxstd=17"
+    - "root@6.28:"
     - "swig@4.1:"
-    - "vecgeom@1.2.4: +gdml cxxstd=17"
+    - "vecgeom@1.2.4: +gdml"
```

Comment:
Would it be too difficult to bump VecGeom version up to 1.2.8?
There are some important bug fixes after 1.2.4.

Reply:
This is just the minimum required to build as specified in the CMake file; Spack will build the newest it has. We have unit tests showing the failures before 1.2.8 in case anyone *does* try to build with an older version.